client/cypress/integration/top_half_spec.js

```diff
@@ -1,31 +1,33 @@
-describe('Top Half', () => {
+describe('SearchPanel', () => {
 
-    it('Go button disabled when one of selector is empty', () => {
+	it('Go button should be disabled when some of selector is empty', () => {
 
-      cy.visit('/')
+		cy.on('uncaught:exception', () => false);
```

Comment:
I don't really know what this command do. As i read the document it tell that "returning false here prevents Cypress from failing the test". But i've tried making the code error by calling some undefined variable, it still fail.

Reply:
Got `ResizeObserver loop limit exceeded` on render while testing, can't reproduce and found out that we can just ignore it.

https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded

Reply:
Ahh, got it.